### PR TITLE
Leaderboard data is now updated when user clicks on refresh leaderboard 

### PR DESCRIPTION
**Pull request by gautamjajoo**
**Issue:** Leaderboard data is not updated when page is outdated and user clicks on refresh leaderboard option

**Fix:** The function was working fine, the label was actually coverred with text hence increased the `z-index` fixes the issue.

https://user-images.githubusercontent.com/24366008/119462107-039b0a80-bd5e-11eb-9323-06a1c9fee656.mp4

@Ram81 @Kajol-Kumari

### CODE REVIEW

File: frontend_v2/src/styles/base.scss [hunk 1]
```diff
@@ -346,6 +346,7 @@ mat-chip {
 
 .update-page {
   display: block;
+  z-index: 1;
```
Comment (Kajol-Kumari) [inline frontend_v2/src/styles/base.scss:349]: Please don't make this change in the base class, instead you should override the same class under the leaderboard component.

Comment (gautamjajoo) [reply frontend_v2/src/styles/base.scss:349]: We are using this class only for the refresh label and we are using this class in two components where we would be facing this problem, that's why I did the changes in `base` class.

Comment (Kajol-Kumari) [reply frontend_v2/src/styles/base.scss:349]: Okay, so if we require this change at both places, it's totally fine to change it in here.

**Review by Kajol-Kumari — APPROVED** [review]
LGTM 👍

**Review by Ram81 — APPROVED** [review]
LGTM

**Pull request merged**